CSS/modules/button.css

```diff
@@ -0,0 +1,18 @@
+.main-body__btn__container {
```

Comment:
let button. css only have button related code, not the wrapper on it and name things simple like btn, btn--previous, btn--next